src/ConfigEditor/AuthTypeSwitch.tsx

```diff
@@ -52,13 +56,32 @@ export function AuthTypeSwitch({ useManagedSecret, onChangeAuthType }: Props) {
       }
     >
       <RadioButtonGroup
+        className={styles.buttonGroup}
         options={[
           { label: 'Temporary credentials', value: false },
           { label: 'AWS Secrets Manager', value: true },
         ]}
         value={useManagedSecret}
         onChange={onChangeAuthType}
+        fullWidth
       />
     </Label>
   );
 }
+
+const getStyles = (theme: GrafanaTheme2) => ({
```

Comment:
These adjustments don't change the old form, only the new form with width adjustments.